addons/skin.estuary/xml/Includes_Controllers.xml

```diff
@@ -207,4 +207,137 @@
 			</control>
 		</control>
 	</include>
+	<include name="GameDialogPorts">
+		<control type="group">
+			<centertop>50%</centertop>
+			<centerleft>50%</centerleft>
+			<width>1820</width>
+			<height>830</height>
+			<include content="DialogBackgroundCommons">
+				<param name="width" value="1820" />
+				<param name="height" value="830" />
+				<param name="header_label" value="" />
+				<param name="header_id" value="2" />
+			</include>
+			<control type="group">
+				<description>Content area</description>
+				<top>100</top>
+				<bottom>30</bottom>
+				<left>30</left>
+				<right>30</right>
+				<!--
+				<control type="image">
+					<description>Content area background for testing</description>
+					<top>-20</top>
+					<bottom>-20</bottom>
+					<left>-20</left>
+					<right>-20</right>
+					<texture border="40">buttons/dialogbutton-nofo.png</texture>
+				</control>
+				-->
```

Comment:
Intended for this to be going in?

Reply:
I used it for testing, but it's not needed anymore and can be dropped.